src/hooks/useAccountENSDomains.ts

```diff
@@ -16,29 +13,19 @@ const queryKey = ({ accountAddress }: { accountAddress: string }) => [
   accountAddress,
 ];
 
-const imagesQueryKey = ({ name }: { name: string }) => ['domainImages', name];
-
 const STALE_TIME = 10000;
 
 async function fetchAccountENSDomains({
   accountAddress,
 }: {
   accountAddress: string;
 }) {
-  if (!accountAddress) return [];
   const result = await fetchAccountRegistrations(accountAddress);
   const registrations = result.data?.account?.registrations || [];
-  const domains = await Promise.all(
-    registrations.map(async ({ domain }) => {
-      const images = await fetchAccountENSImages(domain.name);
-      return {
-        ...domain,
-        images,
-      };
-    })
-  );
-
-  return domains;
+  return registrations.map(({ domain }) => {
+    prefetchENSAvatar(domain.name, { cacheFirst: true });
```

Comment:
`cacheFirst` means that it will use the avatar in local storage (if exists), otherwise it will perform network request